src/commands/functions-artifacts-setpolicy.ts

```diff
@@ -26,18 +26,13 @@ export const command = new Command("functions:artifacts:setpolicy")
   )
   .option(
     "--days <days>",
-    "Number of days to keep container images before deletion. Default is 3 days.",
-    "3",
+    `Number of days to keep container images before deletion. Default is ${artifacts.DEFAULT_CLEANUP_DAYS} day.`,
+    `${artifacts.DEFAULT_CLEANUP_DAYS}`,
```

Comment:
I missed this in another PR, but can we actually tell the difference between getting a default value and no value here? In other words is there a way to know the difference between `--days 1 --none` and `--none`?

Reply:
Fixed it so that we can differentiate the 2 states.